### PR TITLE
typos

### PR DESCRIPTION
**Pull request by crStiv**

### CODE REVIEW

File: .github/workflows/nightly_build.yml
Comment (tersec) [inline .github/workflows/nightly_build.yml:1]: Why does it remove this file?

File: beacon_chain/consensus_object_pools/block_pools_types.nim [hunk 1]
```diff
@@ -139,7 +139,7 @@ type
       ## of each potential head block in this table.
 
     genesis*: Opt[BlockId]
-      ## The root of the genesis block, iff it is known (ie if the database was
+      ## The root of the genesis block, if it is known (ie if the database was
```
Comment (tersec) [inline beacon_chain/consensus_object_pools/block_pools_types.nim:142]: https://en.wikipedia.org/wiki/If_and_only_if

File: .github/workflows/nightly_build.yml
Comment (crStiv) [reply .github/workflows/nightly_build.yml:1]: I accidentally pushed the file "multiple typos "with changes "artefact to artifact" and maybe you can accept all the other typos and decline this wrong one?

Comment (crStiv) [reply .github/workflows/nightly_build.yml:1]: Or I can just redo this PR

Comment (tersec) [reply .github/workflows/nightly_build.yml:1]: > I accidentally pushed the file "multiple typos "with changes "artefact to artifact" and maybe you can accept all the other typos and decline this wrong one?

Not how Git works

**Pull request closed without merging**